test/util.js

```diff
@@ -1,5 +1,3 @@
-global.window = global;

```

Comment:
why this?

Reply:
Same as [here](https://github.com/ripe-tech/ripe-commons-js/pull/33#discussion_r856304572)